client/src/components/buttons.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { thumbsUp, thumbsDown } from '../svg.js';
+import { Helpful, ButtonStyle, ButtonContainer, Thumbs, Report } from '../styles/buttons.style.js';
+
+const Buttons = () => (
+  <div>
+    <Helpful>Was this review helpful?</Helpful>
+    <ButtonStyle>
+      <ButtonContainer>
+        <Thumbs className="thumbs-up" value="yes">
+          <span dangerouslySetInnerHTML={{ __html: thumbsUp }}></span>
```

Comment:
Is this to get the thumbsUp emoji?

Reply:
yes. Importing the SVG path from my svg.js file.